libs/payments/stripe/src/lib/stripe.manager.ts

```diff
@@ -63,6 +63,35 @@ export class StripeManager {
     return customer;
   }
 
+  /**
+   * Updates a customer record
+   */
+  async updateCustomer(
+    customerId: string,
+    params?: Stripe.CustomerUpdateParams
+  ) {
+    return await this.client.customersUpdate(customerId, params);
+  }
+
+  /**
+   * Create customer stub account
+   */
+  async createPlainCustomer(args: { email?: string; taxAddress?: TaxAddress }) {
+    if (args.taxAddress) {
+      return this.client.customersCreate({
+        shipping: {
+          name: args.email || '',
+          address: {
+            country: args.taxAddress.countryCode,
+            postal_code: args.taxAddress.postalCode,
+          },
+        },
+      });
+    }
+
+    return this.client.customersCreate();
```

Comment:
If taxAddress isn't passed, then email also isn't set on the customer.

Reply:
Additionally looks like email is just being set on the customer's name field, but should be set upon the customer itself as the email field.